app/assets/javascripts/pages/Show.js

```diff
@@ -0,0 +1,83 @@
+const getShows = async (id) => {
+     const options = {
+        method: 'GET',
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    };
+    try {
+        const response = await fetch(`http://localhost:3000/interviews/` + id, options)
+        const json = await response.json();
+        return json
+    } catch (err) {
+        console.log('Error getting documents', err)
+        return null
+    }
+}
+
+function myFunction(id) {
+  var xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+    if (this.readyState == 4 && this.status == 200) {
+      document.getElementById("demo").innerHTML = "Interview successfully deleted.";
+    }
+  };
+  xhttp.open("DELETE", "/interviews/" + id, true);
+  xhttp.send();
```

Comment:
Why Xml format is used here? Any specific reason?

Reply:
For calling delete method from controller.